tests/yam/agama/auto/playwright_test.pm

```diff
@@ -0,0 +1,22 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run playwright tests directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi qw(assert_screen assert_script_run get_var select_console);
+
+sub run {
+    my $testcase = get_var('PLAYWRIGHT_TESTCASE');
+
+    assert_screen('agama_product_selection', 120);
+    $testapi::password = 'linux';
+    select_console 'root-console';
+    assert_script_run("SCREENSHOT_MODE=1 RUN_INSTALLATION=1 playwright test --trace on --project chromium --config /usr/share/agama-playwright $testcase", timeout => 1200);
```

Comment:
we don't need those CI screenshot, you can drop that setting, please upload the trace from SUT to openQA once finished.

Reply:
we are missing to reboot the system.

Reply:
Where is the trace located?  And do you mean we need reboot system after the test?

Reply:
actually I don't know where it is, probably under `/usr/share/agama-playwright/...`, but I could search that trace.zip should be the default name.
Of course we need to reboot, we should end up in the new system.